### ISSUE TITLE
Test accuracy remain at 0.65

### ISSUE THREAD

**Issue by donovanOng** [songyouwei/ABSA-PyTorch#46]
After cloning, I ran the following command 

`python train.py --model_name bert_spc --dataset restaurant --learning_rate 2e-5 --logdir bert_spc_logs`

```
epoch:  10
loss: 0.9447, acc: 0.6562, test_acc: 0.6500, f1: 0.2626
loss: 0.9312, acc: 0.6562, test_acc: 0.6500, f1: 0.2626
loss: 1.0388, acc: 0.6042, test_acc: 0.6500, f1: 0.2626
loss: 0.9838, acc: 0.5977, test_acc: 0.6500, f1: 0.2626
loss: 0.9509, acc: 0.6000, test_acc: 0.6500, f1: 0.2626
loss: 0.9240, acc: 0.6068, test_acc: 0.6500, f1: 0.2626
loss: 0.8728, acc: 0.6205, test_acc: 0.6500, f1: 0.2626
loss: 1.0066, acc: 0.6113, test_acc: 0.6500, f1: 0.2626
loss: 0.9481, acc: 0.6111, test_acc: 0.6500, f1: 0.2626
loss: 0.9346, acc: 0.6125, test_acc: 0.6500, f1: 0.2626
loss: 0.9435, acc: 0.6122, test_acc: 0.6500, f1: 0.2626
```
The test accuracy remains at 0.65 and doest not change.

**Comment by GeneZC** [comment]
Similar to #38 .

**Comment by songyouwei** [comment]
Nice issue!
This is a bug caused by [this commit](https://github.com/songyouwei/ABSA-PyTorch/commit/e6f4d59410e11db4535d5165d6e2527946de77aa).
